src/PSRule/Rules/RuleFilter.cs

```diff
@@ -95,14 +94,18 @@ private bool IsIncluded(string name, ResourceTags tag)
             return false;
         }
 
-        private bool MatchWildcard(string name)
+        private bool Contains(string name, string[] set)
         {
-            if (_WildcardMatch == null)
-            {
-                return false;
-            }
+            for (var i = 0; set != null && i < set.Length; i++)
+                if (ResourceIdEqualityComparer.IdEquals(name, set[i]))
+                    return true;
```

Comment:
I wonder if we can use LINQ here: `set.Any(item => ResourceIdEqualityComparer.IdEquals(name, item))` here? Unless we are avoiding LINQ for performance impact.

Reply:
Yep. Faster and less memory is better for low complexity solutions. But good suggestion.